src/components/user-form/AddUserForm.js

```diff
@@ -33,14 +37,26 @@ class AddUserForm extends Component {
     }
 
     console.log(firstName, lastName, email);
-    // todo 1: использовать функцию onUserAdd для создания нового пользователя здесь
+    // done_todo 1: использовать функцию onUserAdd для создания нового пользователя здесь
     //  у этого объекта должны быть проперти first_name, last_name, address, email
-
-    // todo 3: зачистить форму, вызвав функцию onReset
+  onUserAdd({
+    firstName,
```

Comment:
Key should be different
for example firstName should be first_name

Please check if user is added to users list when you press submit